src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -1326,7 +1330,9 @@ public void onPrefsLoaded(SharedPreferences preferences) {
     /* package */ UpdatesFromMixpanel constructUpdatesFromMixpanel(final Context context, final String token) {
         if (Build.VERSION.SDK_INT < MPConfig.UI_FEATURES_MIN_API) {
             Log.i(LOGTAG, "Web Configuration, A/B Testing, and Dynamic Tweaks are not supported on this Android OS Version");
-            return new UnsupportedUpdatesFromMixpanel(sSharedTweaks);
+            return new NoOpUpdatesFromMixpanel(sSharedTweaks);
+        } else if (mConfig.getDisableViewCrawler()) {
```

Comment:
I think having them in the same clause is clear enough - the `||` explains everything. Or, you can have:
```
if (Build.VERSION.SDK_INT < MPConfig.UI_FEATURES_MIN_API ||
     mConfig.getDisableViewCrawler()) {...}
```
to make it clearer.
Having two blocks return the same thing looks weird to me.